### PR TITLE
Famous systems

### PR DESCRIPTION
**Pull request by yuxiliu1995**
Added Jacobians and documentation for systems in `continuous_famous_systems.jl`, and added a historically important chaotic system: [Ueda's attractor](http://scholarpedia.org/article/Attractor).

### CODE REVIEW

File: src/continuous_famous_systems.jl [hunk 1]
```diff
@@ -484,17 +499,31 @@ nosehoover(u0 = [0, 0.1, 0])
 \\dot{z} &= 1 - y^2
 \\end{aligned}
 ```
-Three dimensional conservative continuous system, taken from the book
-"Elegant Chaos" by J. C. Sprott.
+Three dimensional conservative continuous system, discovered in 1984 during
+investigations in thermodynamical chemistry by Nosé and Hoover, then
+rediscovered by Sprott during an exhaustive search as an extremely simple
+chaotic systems. [1]
```
Comment (Datseris) [inline src/continuous_famous_systems.jl]: systems -> system

File: src/continuous_famous_systems.jl [hunk 2]
```diff
@@ -613,3 +654,46 @@ function _Uxx(x, y, c, d0, r)
...
+\\ddot{x} + k \\dot{x} + x^3 = B\\cos{t}
+```
+Nonautonomous Duffing-like forced oscillation system, discovered by Ueda in
+1961. It is one of the first chaotic systems to be discovered.
+
+The stroboscopic plot in the (x, ̇x) plane with period 2π creates a "broken-egg
+attractor" for k = 0.1 and B = 12. Figure 5 of [1] is reproduced by
+
+```julia
+ds = Systems.ueda()
```
Comment (Datseris) [inline src/continuous_famous_systems.jl]: Maybe we should add `using Plots` here

**Comment by Datseris** [comment]
Awesome, thanks! Just some minor comments and then we merge it away!

**Comment by Datseris** [comment]
Ah sorry @yuxiliu1995 I forgot: can you please increment the version in `Project.toml` so that we release these changes immediatelly?

**Comment by yuxiliu1995** [comment]
Can you do it after merging? I don't know how to deal with versioning.

**Comment by Datseris** [comment]
Its actually very straightfoward: see my latest commit!

**Pull request merged**